eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -65,7 +65,15 @@ parameters:
   type: string
   default: ''
 
-- name: targetRid
+- name: runTests
```

Comment:
I wonder if we shouldn't name this to help differentiate it so that the reader doesn't think this is in-regards to the product repo tests?  e.g. runIntegrationTests?  This rename could be considered beyond the scope of your changes as it is more future looking.